miso-service/src/main/resources/log4j.properties

```diff
@@ -0,0 +1,27 @@
+#
```

Comment:
I think it's something I added to try to get my test suite working again. Will remove.